.github/workflows/smoke-test.yml

```diff
@@ -0,0 +1,43 @@
+name: smoke-test
+
+on:
+  push:
```

Comment:
Would it be too annoying or slow to also run this on all pull requests? I often miss breaking builds that only run on main.

Reply:
i think it shouldn't be too slow to run it on every pr. My only concern is we may calling sigstore repo too often, but it should be fine, right? @bdehamer what do you think?

Reply:
Don't think we need to worry about this as the TUF files are served directly from google's cloud storage, which presumably can handle practically infinite request load 😄